src/onboarding/Start.js

```diff
@@ -248,21 +268,24 @@ class StartContent extends React.PureComponent {
                   </Status>
                 </Field>
 
-                <span style={{ height: '40px' }}>
+                <SubmitWrap>
                   {domainCheckStatus === DomainCheckAccepted && (
-                    <Button mode="outline" compact onClick={onOpenOrganization}>
+                    <StyledSubmitButton
+                      mode={responsiveLanding ? 'strong' : 'outline'}
+                      compact={!responsiveLanding}
+                      onClick={onOpenOrganization}
+                    >
                       Open organization
```

Comment:
This button’s label is “Next” in the Sketch, I’m not sure if it should be changed here? /cc @jounih

Reply:
I asked @jounih too as the button when you "search" for an organization in desktop reads `Open organization`.

Reply:
I think let's make the page title "Open an organization" like @bpierre suggested, I think the button is simpler as "Next" but I don't mind, whatever you think works